docs/readme.md

```diff
@@ -1 +1,22 @@
-- placeholder -
+## Plugin Install Instructions
+
+
+### Supported IDEs
+
+
+The Flutter Plugin can be installed on following IntelliJ-based products:
+
+* IntelliJ 2016.2+ ([Ultimate or Community](https://www.jetbrains.com/idea/download/))
+
+
+### Installation
```

Comment:
OK.  There are some issues here.  I started to put together a flow to described Dart setup but hit #178.  I'd like to fix that then land another iteration here that has a cleaner flow.